[new] Add leaflet-hash@0.2.1

louh opened the pull request:
Hi, I'd like to add the [leaflet-hash](https://github.com/mlevans/leaflet-hash) library to cdnjs.

Repository: https://github.com/mlevans/leaflet-hash
Website: http://mlevans.github.io/leaflet-hash/
npm package: https://www.npmjs.com/package/leaflet-hash

The package has over 100 stars on GitHub. It's also a commonly used plugin to leaflet, since map companies like Mapbox and Mapzen self-host it for their / our own use. However, it would be quite useful to have it be a part of cdnjs so we can also fetch it along with leaflet.

Note that the original repository does not have a minified version so I created one myself with Uglify.

Thanks!

PeterDaveHello commented on ajax/libs/leaflet-hash/package.json:
```diff
@@ -0,0 +1,27 @@
+    {
+      "basePath": "/",
+      "files": [
+        "*.js"
```
Please use `leaflet-hash*.js` because root is not a dist folder. Should limit the wildcard.

PeterDaveHello commented on ajax/libs/leaflet-hash/package.json:
```diff
@@ -0,0 +1,27 @@
+  },
+  "main": "leaflet-hash.js",
+  "optionalDependencies": {},
+  "engines": {
```
Please remove useless `engines` and `optionalDependencies`.

PeterDaveHello commented:
Please add some keywords for it, thanks.

PeterDaveHello commented:
@Piicksarn please be review this PR.

louh commented:
Done and squashed into previous commit.

- Changed file `*.js` in package.json to `leaflet-hash*.js`
- Removed `engines`, `optionalDependencies`, and `main` from package.json.
- Added keywords.

PeterDaveHello commented:
Thanks @louh. 
Now you can access it via:
https://cdnjs.cloudflare.com/ajax/libs/leaflet-hash/0.2.1/leaflet-hash.js
https://cdnjs.cloudflare.com/ajax/libs/leaflet-hash/0.2.1/leaflet-hash.min.js
